Tests/functional/TransformationSandboxTest.php

```diff
@@ -0,0 +1,259 @@
+    {
+        $configuration = [
+            'backend' => 'docker',
+            'description' => 'Test configuration',
+            'type' => 'r',
+            'input' => [
+                [
+                    'source' => 'in.c-main.source',
+                    'destination' => 'destination.csv',
+                    'whereColumn' => '',
+                    'changedSince' => '',
+                ],
+                [
+                    'source' => 'in.c-main.source',
+                    'destination' => 'filtered.csv',
+                    'whereColumn' => 'id',
+                    'whereOperator' => 'eq',
+                    'whereValues' => ['68640847'],
+                    'changedSince' => '',
+                ],
+            ],
+            'output' => [
+                [
+                    'destination' => 'out.c-main.r-transpose',
+                    'source' => 'transpose.csv',
+                ],
+            ],
+            'packages' => [],
+            'tags' => [],
+            'queries' => [
+                'this is some script\ncode on multiple lines',
+            ],
+        ];
+
+        $vars = $this->createConfiguration($configuration);
+        $envs = [
+            'KBC_TOKEN' => getenv('KBC_TEST_TOKEN'),
+            'KBC_STORAGEAPI_URL' => getenv('KBC_TEST_URL'),
+            'KBC_CONFIG_ID' => $vars['configId'],
+            'KBC_ROW_ID' => $vars['rowId'],
+            'KBC_CONFIG_VERSION' => '2',
+        ];
+        $specification = new DatadirTestSpecification(
+            __DIR__ . '/transformation-sandbox-tables/source/data',
+            0,
+            null,
+            '',
+            __DIR__ . '/transformation-sandbox-tables/expected/data/in'
+        );
+        $tempDatadir = $this->getTempDatadir($specification);
+        $process = $this->runScript($tempDatadir->getTmpFolder(), $envs);
+        $this->assertMatchesSpecification($specification, $process, $tempDatadir->getTmpFolder());
+        $output = $process->getOutput();
+        $output = preg_replace('#\[.*?\]#', '', $output);
+        $output = preg_replace('#\{.*?\}#', '', $output);
+        self::assertEquals(
+            implode("\n", [
+                ' app-logger.INFO: Starting Data Loader  ',
+                ' app-logger.INFO: DataLoader is loading data  ',
+                ' app-logger.INFO: Reading configuration ' . $vars['configId'] . ', row: ' . $vars['rowId'] . '  ',
+                ' app-logger.INFO: Script size 43  ',
+                ' app-logger.INFO: There are no input files.  ',
+                ' app-logger.INFO: Fetched table in.c-main.source.  ',
+                ' app-logger.INFO: Fetched table in.c-main.source.  ',
+                ' app-logger.INFO: Processing 2 table exports.  ',
+                ' app-logger.INFO: All tables were fetched.  ',
+                '',
+            ]),
+            $output
+        );
+        $this->components->deleteConfiguration('transformation', $vars['configId']);
```

Comment:
not sure if it may be better to handle cleanup in setup rather than here.  if the test fails it won't clean.